src/test/java/uk/gov/hmcts/reform/translate/service/DictionaryServiceTest.java

```diff
@@ -120,27 +166,117 @@ private DictionaryEntity createDictionaryEntity(String phrase, String translatio
             .build();
     }
 
-    @Test
-    void shouldReturnEmptyDictionaryContents() {
-        given(dictionaryRepository.findAll()).willReturn(repositoryResults);
-        assertTrue(dictionaryService.getDictionaryContents().isEmpty());
-    }
+    @Nested
+    @DisplayName("GetTranslation")
+    class GetTranslations {
 
-    @Test
-    void shouldThrowExceptionWhenReturningDictionaryContentsNoUserInfoAvailable() {
-        Mockito.reset(securityUtils);
-        assertThrows(RoleMissingException.class, () -> dictionaryService.getDictionaryContents());
-    }
+        @Test
+        void testShouldTranslatePhrase() {
+            final DictionaryEntity dictionaryEntity = createDictionaryEntity(THE_QUICK_FOX_PHRASE, "translated");
+            doReturn(Optional.of(dictionaryEntity)).when(dictionaryRepository).findByEnglishPhrase(anyString());
+
+            final String translation = dictionaryService.getTranslation(THE_QUICK_FOX_PHRASE);
+
+            assertThat(translation)
+                .isNotNull()
+                .isEqualTo("translated");
+
+            verify(dictionaryRepository).findByEnglishPhrase(eq(THE_QUICK_FOX_PHRASE));
```

Comment:
Can you resolve the sonar issues flagged against this file.

Reply:
Fixed in 4061f2f5158e57ec732f5b040ac1c0f317bade8a